### ISSUE TITLE
Armor Plates Not being put on in server. Modlist

### ISSUE THREAD

**Issue by Wispyee** [ClumsyAlien/TimelessandClassics_Reforged#142]
Patreon Alpha Version 1.18.2

When applying the Armor Repair Plate, It doesn't apply to the body armor. 

I've attempted this both on a server and in a single player world. 

If there is something in the config I could adjust my end or a mod that is clashing with it I'd love to know. 

Really love your guys work. The best gun mod I've ever played with. 

[Mods.txt](https://github.com/ClumsyAlien/TimelessandClassics_Reforged/files/12744167/Mods.txt)

**Comment by Type-32** [comment]
yeah this happens in 1.18.2 alpha version of the mod. It'll probably be fixed in the next 1.18.2 version release
